docs/framework/configure-apps/file-schema/wcf/servicediscovery.md

```diff
@@ -27,7 +27,7 @@ Specifies the discoverability of service endpoints.
 ## Syntax  
   
 ```  
-<behaviors>  <serviceBehaviors>    <behavior name=String">      <serviceDiscovery>        <announcementEndpoints>              <endpoint name="String”                        kind="Type" />        </announcementEndpoints>        <discoveryEndpoints>              <endpoint name="String”                        kind="Type" />        </discoveryEndpoints>      </serviceDiscovery>    </behavior>  </serviceBehaviors></behaviors>  
+<behaviors>  <serviceBehaviors>    <behavior name=String">      <serviceDiscovery>        <announcementEndpoints>              <endpoint name="String"                        kind="Type" />        </announcementEndpoints>        <discoveryEndpoints>              <endpoint name="String"                        kind="Type" />        </discoveryEndpoints>      </serviceDiscovery>    </behavior>  </serviceBehaviors></behaviors>  
```

Comment:
There's quite a lot of XML code blocks formatted terribly like this one.

They will need to be fixed in a separate PR.